### PR TITLE
Point/fix multiple bugs

### PR DESCRIPTION
**Pull request by pointmekin**
# Notes
Fixed and tested multiple small-medium sized general bugs assigned to "Point" as reported in https://docs.google.com/spreadsheets/d/1pS3LS3CGNMryoAY_DO0kaq3hxtRGYR0818xuLeQcdh8/edit?usp=sharing

[There's quite a lot and they come form various pages so I can't list them all here lol]

### CODE REVIEW

File: src/contexts/HealthRecordContext.tsx [hunk 1]
```diff
@@ -45,14 +47,15 @@ const HealthRecordProvider = ({ ...props }) => {
...
-    const tempElderlyUid = currentElderlyUid;
+    const tempElderlyUid = elderlyUid;
     const { data } = await client.get(
       `healthRecord/table/${
         user.isElderly ? '' : `${tempElderlyUid}/`
       }${tempHrName}`
     );
     const _data = { ...data, data: data.data.reverse() };
     setHealthTable(_data);
+    console.log(_data);
```
Comment (bweam) [inline src/contexts/HealthRecordContext.tsx]: remove this?

File: src/screens/healthRecord/AddHealthEntryScreen.tsx [hunk 2]
```diff
@@ -88,6 +91,11 @@ const AddHealthEntry = () => {
       data: fields
     };
     try {
+      console.log(
+        `healthRecord/healthData/${user?.isElderly ? 'elderly' : 'caretaker'}/${
+          user.isElderly ? '' : currentElderlyUid
+        }`
+      );
```
Comment (bweam) [inline src/screens/healthRecord/AddHealthEntryScreen.tsx]: this too

**Review by bweam — APPROVED** [review]
LGTM

File: src/contexts/HealthRecordContext.tsx (hunk 1 above)
Comment (pointmekin) [reply src/contexts/HealthRecordContext.tsx]: Oops. Forgot to remove. Thxx

**Pull request merged**